### DISCUSSION TITLE
Can't find Tabletennis live odds

### DISCUSSION THREAD

**Discussion by RomainTess** [cloudbet/docs#81] [Q&A]
Hello, I am relatively new to the programming world and I was testing out the API on the documentation, trying to retreive some live tabletennis odds. The response I get is quite weird, as if the page doesn't exist:

cache-control: max-age=0,no-store 
 content-length: 0 
 
 Could anyone help me out with this please? Would be really appreciated

**Comment by kgravenreuth** [comment]
Hi @RomainTess ,

Could you please share more information about your request, for example which API endpoint you were trying to hit exactly and the http response status code? If you could share a `curl` request example, then it would be helpful for us to debug further. 

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by RomainTess** [reply]
Hi, just figured it out. Had the wrong spelling for the sports, that was all. Also, any chance you have an update on upcoming feature for parlays?

**Reply by kgravenreuth** [reply]
Hey, glad you figured out the issue with table tennis odds. 👍 

We don't have any updates on parlays in the Cloudbet API yet. We'll definitely update this discussions forum when we have feature updates.
